components/DsmAirdrop/index.tsx

```diff
@@ -151,6 +151,9 @@ const DsmAirdrop: React.FC = () => {
               }}
               amount={totalDsmAllocated}
               chainConnections={chainConnections}
+              onConnectChains={() => {
+                setStage(CommonStage.ConnectChainsStage)
```

Comment:
if we switch to connect chains page then we need to add a "Claim" button on the page as well. otherwise users will be stuck at the connect chain page